changelog/2.26.0.md

```diff
@@ -42,3 +42,13 @@ Released 2024-11-08
 - [4ecbee6](https://github.com/elastisys/kubespray/commit/4ecbee6c0f245a6ba0fb33eba36bfd97e5211c14) - terraform upcloud: split LB proxy protocol config per backend @davidumea
 - [d40faef](https://github.com/elastisys/kubespray/commit/d40faef54f47ff26442ff271e6ec01b251d948d9) - terraform upcloud: fix flexible plans @davidumea
 - [4113920](https://github.com/elastisys/kubespray/commit/4113920ad99e7f1853c3c215534375bd43766b45) - upcloud: encrypted at rest volumes @robinAwallace
+
+## v2.26.0-ck8s2
```

Comment:
Is it the script that puts this last in the file or us via the process?

I would prefer if newer patches was at the top, as they are shown in github release list.

Reply:
Seems to be the script that does this https://github.com/elastisys/compliantkubernetes-kubespray/blob/main/release/stage-release.sh#L52-L54

Reply:
Yeah, it's the script. I didn't make any manual changes to the changelog.